students/jeff_shabani/session05/mailroom_3.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+from collections import OrderedDict
+from operator import itemgetter
+from pathlib import Path
+
+DONORS = {'William B': [120, 130, 50],
+          'Sammy Maudlin': [500, 125, 670, 1000],
+          'Bobby Bittman': [10],
+          'Skip Bittman': [75, 125, 19],
+          'Ashley Lashbrooke': [10000, 15000]}
+
+prompt = "\n".join(("Welcome to my charity!",
+                    "Please select and option below:",
+                    "1 - Send a Thank You to an individual",
+                    "2 - Create a Report",
+                    "3 - Send letters to all donors",
+                    "4 - Quit",
+                    ">>> "))
+
+def value_error():
+    """
+    Catch non-numeric entries for donation amount.
+    """
+    amount = int()
```

Comment:
this should be `amount = None`

Reply:
Why should it be None?

Reply:
convention.  When initializing a value in python we usually set to None rather than a type.  notice:
```>>> amount = int()
>>> amount
0
```
amount is a value here.  That could break logic somewhere if its not expected to be.

Reply:
Where as
```
>>> amount = None
>>> amount
>>> 
```